infra/powershell/Join-Domain.ps1

```
@@ -0,0 +1,63 @@
+    [string]$TargetHost,
+
+    [Parameter(Mandatory)]
+    [string]$ProfileCode,
+
+    [string]$DomainName = "",
+
+    [string]$OUPath = "",
+
+    [string]$DomainAdmin = "",
+
+    [string]$DomainAdminPassword = ""
```

PSScriptAnalyzer (1.25.0)

[warning] Parameter '$DomainAdminPassword' should not use String type but either SecureString or PSCredential, otherwise it increases the chance to expose this sensitive information.

Suggested fix: Set $DomainAdminPassword type to SecureString; Set $DomainAdminPassword type to PSCredential

(PSAvoidUsingPlainTextForPassword)